lib/supavisor/helpers.ex

```diff
@@ -88,7 +88,9 @@ defmodule Supavisor.Helpers do
   @spec get_user_secret(pid(), String.t(), String.t()) :: {:ok, map()} | {:error, String.t()}
   def get_user_secret(conn, auth_query, user) do
     try do
-      Postgrex.query!(conn, auth_query, [user])
+      # Postgrex.query!(conn, auth_query, [user])
+      auth_query = String.replace(auth_query, "$1", "'#{user}'")
```

Comment:
isn't this a potentially danger for sql injection?

Reply:
I'll have to check, but good call!